src/pcrlock/pcrlock.c

```diff
@@ -0,0 +1,5208 @@
+                        return 0;
+
+                return log_error_errno(errno, "Failed to remove old policy file '%s': %m", path);
+        }
+
+        log_info("Removed policy file '%s'.", path);
+        return 0;
+}
+
+static int help(int argc, char *argv[], void *userdata) {
+        _cleanup_free_ char *link = NULL;
+        int r;
+
+        r = terminal_urlify_man("systemd-pcrlock", "1", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%1$s  [OPTIONS...] COMMAND ...\n"
+               "\n%5$sManage a TPM2 PCR lock.%6$s\n"
+               "\n%3$sCommands:%4$s\n"
+               "  log                         Show measurement log\n"
+               "  cel                         Show measurement log in TCG CEL-JSON format\n"
+               "  list-components             List defined .pcrlock components\n"
+               "  predict                     Predict PCR values\n"
+               "  make-policy                 Predict PCR values and generate TPM2 policy from it\n"
+               "  remove-policy               Remove TPM2 policy\n"
+               "\n%3$sProtections:%4$s\n"
+               "  lock-firmware-code          Generate a .pcrlock file from current firmware code\n"
+               "  unlock-firmware-code        Remove .pcrlock file for firmware code\n"
+               "  lock-firmware-config        Generate a .pcrlock file from current firmware configuration\n"
+               "  unlock-firmware-config      Remove .pcrlock file for firmware configuration\n"
+               "  lock-secureboot-policy      Generate a .pcrlock file from current SecureBoot policy\n"
+               "  unlock-secureboot-policy    Remove .pcrlock file for SecureBoot policy\n"
+               "  lock-secureboot-authority   Generate a .pcrlock file from current SecureBoot authority\n"
+               "  unlock-secureboot-authority Remove .pcrlock file for SecureBoot authority\n"
+               "  lock-gpt [DISK]             Generate a .pcrlock file from current GPT header\n"
+               "  unlock-gpt                  Remove .pcrlock file for GPT header\n"
+               "  lock-pe [BINARY]            Generate a .pcrlock file from PE binary\n"
+               "  unlock-pe                   Remove .pcrlock file for PE binary\n"
+               "  lock-uki [UKI]              Generate a .pcrlock file from UKI PE binary\n"
+               "  unlock-uki                  Remove .pcrlock file for UKI PE binary\n"
+               "  lock-machine-id             Generate a .pcrlock file from current machine ID\n"
+               "  unlock-machine-id           Remove .pcrlock file for machine ID\n"
+               "  lock-file-system [PATH]     Generate a .pcrlock file from current root fs + /var/\n"
+               "  unlock-file-system [PATH]   Remove .pcrlock file for root fs + /var/\n"
+               "  lock-kernel-cmdline [FILE]  Generate a .pcrlock file from current kernel command line\n"
```

Comment:
When FILE is given, "current" is not true. So maybe say "Generate a .pcrlock file from a kernel command line"